en/administration/centreon-ha/acceptance-guide.md

```diff
@@ -0,0 +1,548 @@
+---
+id: acceptance-guide
+title: Cluster acceptance guide
+---
+
+> Please note that all commands presented in this document are respectively to be run as `root` unless otherwise stated.
+
+> This document will refer to parameters that vary from one installation to another (e.g., names and IP addresses of nodes) via. [macros defined here](../../installation/installation-of-centreon-ha/installation-2-nodes.html#defining-names-and-IP-addresses-of-servers)
+
+## Requirements of the tests
+
+To verify the proper functioning of your cluster, you will get all the commands to perform a failover test and simulate network crashes.
+
+It is necessary to check the state of the cluster before performing the acceptance tests. 
+
+### Check the state of the cluster
+
+To check the general state of the cluster, run the command:
+
+```bash
+pcs status
+```
+
+The command should return the following information:
+
+```bash
+Stack: corosync
+Current DC: @CENTRAL_MASTER_NAME@ (version 1.1.23-1.el7_9.1-9acf116022) - partition with quorum
+Last updated: Fri Jul  9 11:09:30 2021
+Last change: Fri Jul  9 11:08:57 2021 by root via crm_resource on @CENTRAL_MASTER_NAME@
+
+2 nodes configured
+14 resource instances configured
+
+Online: [ @CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@ ]
+
+Active resources:
+
+ Master/Slave Set: ms_mysql-master [ms_mysql]
+     Masters: [ @CENTRAL_MASTER_NAME@ ]
+     Slaves: [ @CENTRAL_SLAVE_NAME@ ]
+ Clone Set: php7-clone [php7]
+     Started: [ @CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@ ]
+ Clone Set: cbd_rrd-clone [cbd_rrd]
+     Started: [ @CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@ ]
+ Resource Group: centreon
+     vip        (ocf::heartbeat:IPaddr2):       Started @CENTRAL_MASTER_NAME@
+     http       (systemd:httpd24-httpd):        Started @CENTRAL_MASTER_NAME@
+     gorgone    (systemd:gorgoned):     Started @CENTRAL_MASTER_NAME@
+     centreon_central_sync      (systemd:centreon-central-sync):        Started @CENTRAL_MASTER_NAME@
+     cbd_central_broker (systemd:cbd-sql):      Started @CENTRAL_MASTER_NAME@
+     centengine (systemd:centengine):   Started @CENTRAL_MASTER_NAME@
+     centreontrapd      (systemd:centreontrapd):        Started @CENTRAL_MASTER_NAME@
+     snmptrapd  (systemd:snmptrapd):    Started @CENTRAL_MASTER_NAME@
+```
+
+> Check the resources for `Failed` errors and correct them with the help of the [troubleshooting guide](troubleshooting-guide.html).
```

Comment:
Link is not good here ?

Reply:
Nope. It's good because, is in same directory